### PR TITLE
UI banner & chakra theme

### PR DESCRIPTION
**Pull request by jjmakes**
Created Chakra theme with Levl color palette & fonts. Created basic banner & two pages (HCP & Provider Dashboard)

### CODE REVIEW

**Review by Jayhawk416 — APPROVED** [review]
Everything looks as it should, this is ok to merge into main

File: web/src/Routes.js [hunk 1]
```diff
@@ -6,12 +6,16 @@
 //
 // 'src/pages/HomePage/HomePage.js'         -> HomePage
 // 'src/pages/Admin/BooksPage/BooksPage.js' -> AdminBooksPage
-
-import { Router, Route } from '@redwoodjs/router'
+import { Router, Route, Set } from '@redwoodjs/router'
+import BannerLayout from './layouts/BannerLayout/BannerLayout'
```
Comment (iambillmccann) [inline web/src/Routes.js:10]: You should use 'src/layouts/BannerLayout/BannerLayer' ... importing from the src folder is a Redwood best practice.

File: web/src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+const Navbar = () => {
+  return (
+  <>
+      <Flex alignItems="center">
+        <Box>
+          <img src='images/logo.png' alt='Logo' />
+        </Box>
+        <Spacer />
+        <Box>
+          <ButtonGroup>
```
Comment (iambillmccann) [inline web/src/components/Navbar.js:23]: The "menu item" links in the Figma design are placeholders. We will not be implementing any features for these in the capstone. You can remove all of these including the dropdown.

File: web/src/layouts/BannerLayout/BannerLayout.stories.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import BannerLayout from './BannerLayout'
```
Comment (iambillmccann) [inline web/src/layouts/BannerLayout/BannerLayout.stories.js:1]: Import from the src folder. VSCode intellisense will find the folder for you. Make sure you have the prettier and linting installed and on.

File: web/src/layouts/BannerLayout/BannerLayout.test.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { render } from '@redwoodjs/testing/web'
+
+import BannerLayout from './BannerLayout'
```
Comment (iambillmccann) [inline web/src/layouts/BannerLayout/BannerLayout.test.js:3]: Import from the src folder. VSCode intellisense will find the folder for you. Make sure you have the prettier and linting installed and on.

File: web/src/pages/HcpDashboardPage/HcpDashboardPage.stories.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import HcpDashboardPage from './HcpDashboardPage'
```
Comment (iambillmccann) [inline web/src/pages/HcpDashboardPage/HcpDashboardPage.stories.js:1]: Import from the src folder. VSCode intellisense will find the folder for you. Make sure you have the prettier and linting installed and on.

File: web/src/pages/HcpDashboardPage/HcpDashboardPage.test.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import { render } from '@redwoodjs/testing/web'
+
+import HcpDashboardPage from './HcpDashboardPage'
```
Comment (iambillmccann) [inline web/src/pages/HcpDashboardPage/HcpDashboardPage.test.js:3]: Import from the src folder. VSCode intellisense will find the folder for you. Make sure you have the prettier and linting installed and on.

File: web/src/pages/ProviderDashboardPage/ProviderDashboardPage.test.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import { render } from '@redwoodjs/testing/web'
+
+import ProviderDashboardPage from './ProviderDashboardPage'
```
Comment (iambillmccann) [inline web/src/pages/ProviderDashboardPage/ProviderDashboardPage.test.js:3]: Import from the src folder. VSCode intellisense will find the folder for you. Make sure you have the prettier and linting installed and on.

**Review by iambillmccann — CHANGES_REQUESTED** [review]

**Comment by iambillmccann** [comment]
It looks like CRLF have been added to line endings in at least some of these files (I did not check every one). Is someone developing in Windows? Regardless, be sure to set line ending to LF (and **not** CRLF).

File: web/src/App.js [hunk 8]
```diff
@@ -1,6 +1,9 @@
 import { FatalErrorBoundary, RedwoodProvider } from '@redwoodjs/web'
 import { RedwoodApolloProvider } from '@redwoodjs/web/apollo'
 
+import { ChakraProvider, ColorModeScript, extendTheme } from '@chakra-ui/react'
```
Comment (iambillmccann) [inline web/src/App.js:4]: extendTheme is not used, you should remove the import.

**Review by iambillmccann — CHANGES_REQUESTED** [review]

**Review by iambillmccann — APPROVED** [review]

**Pull request merged**